docs/manual/src/python/setup.md

```diff
@@ -0,0 +1,236 @@
+# Intro
+
+The main idea is to build the bindings with:
+
+python .\setup.py bdist_wheel --verbose ;
+$wheelFile = Get-ChildItem -Path .\dist\ -Recurse -Include * ;
```

Comment:
this is a powershell command which is no use in a cross-platform tool

Reply:
can I provide various commands depending of the OS?

Reply:
This is a markdown file, so I think it's OK to just describe what it happening. For actual commands, maybe you can specify the exact output filename via a command-line option, then there's no need to try and find it?